lib/kernel-manager.coffee

```diff
@@ -91,8 +92,7 @@ class KernelManager
 
 
     getAllKernelSpecs: ->
-        kernelSpecs = _.map @parseKernelSpecSettings(), 'spec'
-        return kernelSpecs
+        return @_kernelSpecs
```

Comment:
We should return a clone, to avoid side-effects. See [getAllRunningKernels](https://github.com/lgeiger/hydrogen/blob/fd07870dce47987d46ea82ea1266d9cba85d4ae3/lib/kernel-manager.coffee#L83)